### PR TITLE
Fixing a bug in incubator alert

### PR DESCRIPTION
**Pull request by lostcoaster**
Fix the bug that incubator alert is not working.

Also add the ability for overlay and log to show icons, like here:

Log
Overlay

(stretching icons a bit with min width 1em, so narrow icons are more noticeable

Also change NodeJS.timer type to be dynamically determined. For some reason on my node 16.17 the type is NodeJS.Timeout and blocking build.

### CODE REVIEW

File: old/main.js [hunk 1]
```diff
@@ -384,12 +384,12 @@ function initWindow(window) {
...
             .map(
               ([name, icon]) =>
-                `<img src='${icon}' class='currencyText'/><span class='eventText'>${name}</span>`
+                `<span><img src='${icon}' class='currencyText'><span class='eventText'>${name}</span></span>`
             )
             .join('; '),
         true
       );
-    }
+    }``
```
Comment (Qt-dev) [inline old/main.js:392]: Should this be here?

**Comment by Qt-dev** [comment]
Oh nice, was the issue only that the setting is not in `activeProfile` anymore then?
Got a small comment about a part I am not sure about but the rest looks solid. Thank you very much!  


I'll include this in the next update. I will try to release a 1.0.3 this week adding this plus any bugfix I can tackle before that.

**Review by Qt-dev — APPROVED** [review]
Oh nice, was the issue only that the setting is not in activeProfile anymore then?
Got a small comment about a part I am not sure about but the rest looks solid. Thank you very much!

I'll include this in the next update. I will try to release a 1.0.3 this week adding this plus any bugfix I can tackle before that.

File: old/main.js (hunk 1 above)
Comment (lostcoaster) [reply old/main.js]: hmm didn't notice that, but as this file was obsoleted I guess it was fine. Will just revert this file

**Comment by Qt-dev** [comment]
You're right. I cleaned up a lot of stuff for that V1 and redid most of the settings' front-end, plugging things more cleanly one by one. I changed some stuff while doing that, getting away from that activeProfile settings a bit. Sorry about that :)

The setup is done [there](https://github.com/Qt-dev/exile-diary/blob/main/src/renderer/components/MainSettings/MainSettings.tsx#L80-L90) if you ever need some more info.

**Pull request merged**